### PR TITLE
base: requirements cleanup

### PR DESCRIPTION
**Pull request by jmartinm**
* Reorganizes requirements and setup.py dependencies.

Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
@jalavik I have cleaned up some dependencies, can you have a look?

Do you know what is ``setuptools-bower``? It does not seem to be used so I removed it.

Also instead of ``dependency_links`` (which was not working for me) I have added the dependencies that need a specific branch to ``requirements.txt``

**Comment by jalavik** [comment]
@jmartinm Cool. `setuptools-bower` is an Invenio thing that would install bower dependencies during Invenio installation: https://github.com/inveniosoftware/setuptools-bower. However, I believe it is not being used anymore. @lnielsen

Seems like `Invenio-Kwalitee` installation is failing. Looks like its called `kwalitee` in PyPi: https://pypi.python.org/pypi/kwalitee

File: setup.py [hunk 1]
```diff
@@ -58,17 +58,13 @@
         "rt",
         "HarvestingKit>=0.3",
         "mixer==4.9.5",
-        "ipython",
-        "Babel>=1.3",
```
Comment (jalavik) [inline setup.py]: I suspect we may still need Babel from a clean installation. If you look at the the `travis.yml` it reveals a few more versioned dependencies we need to add here it seems..

Comment (jalavik) [inline setup.py]: We could try to kill them and see if it is still needed.

**Comment by jalavik** [comment]
@jmartinm Issue with collections. Perhaps this PR should be merged into `master` first?

**Comment by jmartinm** [comment]
The PR was already merged to `master` I think? https://github.com/inspirehep/inspire-next/commits/master

what it the issue with collections?

**Comment by jalavik** [comment]
Oh right. Forgot it was already merged :boom: 

I think we can just close this PR then because at some point we'll rebase next on master I guess?

**Comment by jmartinm** [comment]
Closed.

I am rebasing at the moment so `next` will contain this commit very soon.

**Pull request closed without merging**